.github/workflows/on-pr-label-blog-preview.yml

```
@@ -14,4 +14,6 @@ jobs:
   blog-deploy-preview:
     if: github.event.label.name == 'deploy-preview'
     uses: ./.github/workflows/on-workflow-call-blog-preview-deployment.yml
+    with:
+      add_comment: true
```

actionlint (1.7.4)

input "add_comment" is typed as string by reusable workflow "./.github/workflows/on-workflow-call-blog-preview-deployment.yml". bool value cannot be assigned

(expression)